packages/main/src/Input.js

```diff
@@ -831,6 +833,14 @@ class Input extends UI5Element {
 		};
 	}
 
+	get itemSelectionAnnounce() {
```

Comment:
This code logically belongs to InputSuggestions.js. The clue to this is that you don't actually use the state of Input.js, but refer to this.Suggestions for everything inside. Move it there, and in this method, if suggestions are imported by the user, just return a method call to suggestions